src/Controller/GameController.php

```diff
@@ -27,6 +31,12 @@ public function init(): string
             header("HTTP/1.0 404 Not Found");
             echo '404 - Page not found';
         }
+
+        if (false === $this->enemyManager->deleteAttacker()) {
+            header("HTTP/1.0 404 Not Found");
+            echo '404 - Page not found';
+        }
```

Comment:
You delete attacker never returns false now.
But maybe it should !

And in case of database errors, you should have a 503, not a 404